src/testlib.ts

```diff
@@ -65,22 +119,23 @@ export function validate(intentRequest: object, apiResponse: object, syncData?:
       const attributes = syncDevices[i]['attributes'] || {};
       for (let j = 0; j < traits.length; j++) {
         const trait = traits[j];
-        if (trait in TRAIT_ATTRIBUTES_EXPECT) {
-          const validateTraitRes = responseValidation(attributes, TRAIT_ATTRIBUTES_EXPECT[trait]);
+        const schema = getTraitSchema(trait, 'attributes');
```

Comment:
nit: maybe use a typescript enum for the schema type.

Reply:
Done